src/dbnode/client/session.go

```diff
@@ -2120,13 +2121,17 @@ func (s *session) streamBlocksMetadataFromPeers(
 					firstAttempt = false
 					return true
 				}
-				currLevel := level.value()
-				majority := int(majority)
-				enqueued := int(enqueued)
-				success := int(atomic.LoadInt32(&success))
 
-				doRetry := !topology.ReadConsistencyAchieved(currLevel, majority, enqueued, success) &&
-					errs.getAbortError() == nil
+				var (
+					currLevel          = level.value()
+					majority           = int(majority)
+					enqueued           = int(enqueued)
+					success            = int(atomic.LoadInt32(&success))
+					metReadConsistency = topology.ReadConsistencyAchieved(
```

Comment:
more a pet peeve than anything else: move this and the next line into the (:=) assignment formulation. Calling (non stdlib) functions in a var decl makes is always a bit weird IMO